develop/dev-guide-sample-application-nodejs-typeorm.md

```
@@ -85,7 +85,7 @@ npm install @types/node ts-node typescript --save-dev
 
     -   **接続タイプ**は`Public`に設定されています。
     -   **ブランチ**は`main`に設定されています。
-    -   **「接続」は**`General`に設定されています。
+    -   **「接続」**は`General`に設定されています。
```

markdownlint-cli2 (0.23.2)

[warning] Spaces after list markers
Expected: 1; Actual: 3

(MD030, list-marker-space)